src/ConfigPostProcessor.php

```diff
@@ -251,9 +252,13 @@ private function replaceDependencyConfiguration(array $config)
 
         $config = $this->replaceDependencyInvokables($config);
         $config = $this->replaceDependencyFactories($config);
-        $delegators = isset($config['delegators']) ? $this->replaceDependencyDelegators($config['delegators']) : [];
-        if ($delegators) {
-            $config['delegators'] = $delegators;
+
+        foreach ($config as $key => $data) {
+            if (in_array($key, ['aliases', 'invokables', 'factories'], true)) {
+                continue;
+            }
+
+            $config[$key] = $this->__invoke($config[$key], [$key]);
```

Comment:
we need to check here  - invoke only if `$config[$key]` is array, otherwise .... something else?